src/components/user-group/uc/classes.uc.js

```diff
@@ -0,0 +1,24 @@
+const { classesRepo } = require('../repo');
+
+const getClassesForStudent = async (userId) => {
+	return classesRepo.getClassesForStudent(userId);
+};
+
+const getClassesForTeacher = async (userId) => {
```

Comment:
This should not an exportet uc and should at to the remove uc. The remove value is only the mongoose return and this must changed.